templates/publisher/account-snaps.html

```diff
@@ -0,0 +1,166 @@
+        <p><a href="https://tutorials.ubuntu.com/tutorial/create-your-first-snap#0" class="p-button--neutral p-link--external">Try the tutorial</a></p>
+      </div>
+      <div class="col-6">
+        <h3 class="p-heading--five">Publish a snap with the tools you know</h3>
+        <p>Electron, Python, Go and many more examples of how to 'snap' an application using familiar languages.</p>
+        <p><a href="https://docs.snapcraft.io/build-snaps/languages" class="p-button--neutral">Choose a language guide</a></p>
+      </div>
+    </div>
+  </div>
+  <div class="p-strip is-shallow">
+    <div class="row p-divide">
+      <div class="col-6 p-divider__block">
+        <h3 class="p-heading--five">Snapcraft documentation</h3>
+        <p>All you need to know about snaps, snapcraft and everything in between.</p>
+        <p><a href="https://docs.snapcraft.io">Read the docs &rsaquo;</a></p>
+      </div>
+      <div class="col-6 p-divider__block">
+        <h3 class="p-heading--five">Join the community</h3>
+        <p>Request features, troubleshoot and generally find all the latest talk about snaps, snapd and snapcraft.</p>
+        <p><a href="https://forum.snapcraft.io">Go to the focum &rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+{% else %}
+  <div class="p-strip is-shallow u-no-padding--top">
+    <div class="row u-no-margin--top">
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Privacy</th>
+            <th>Owner</th>
+            <th>Latest release</th>
+            <th>Version</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for snap in snaps|sort %}
+            <tr>
+              <td>
+                <a href="/account/snaps/{{snap}}/market" class="u-no-margin--top u-vertically-center">
+                  {% if snaps[snap].icon_url %}
+                    <img src="{{ snaps[snap].icon_url }}" class="p-snap-list__image" />
+                  {% else %}
+                    <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" class="p-snap-list__image" />
+                  {% endif %}
+                  <span class="p-snap-list__name u-no-margin--top">
+                    {{snap}}
+                  </span>
+                </a>
+              </td>
+              <td>
+                {% if snaps[snap].private %}
+                  Private
+                {% else %}
+                  Public
+                {% endif %}
+              </td>
+              <td>
+                {% if snaps[snap].publisher.username == current_user %}
+                  You
+                {% else %}
+                  {{ snaps[snap].publisher['display-name'] }}
+                {% endif %}
+              </td>
+              {% if snaps[snap].latest_revisions[0].status == 'Published' %}
+                <td>{{ snaps[snap].latest_revisions[0].channels[0] }}</td>
+              {% else %}}
```

Comment:
Additional `}` leaking out of the table:

<img width="1024" alt="screen shot 2018-04-13 at 10 00 17" src="https://user-images.githubusercontent.com/83575/38723490-8b8ad648-3f01-11e8-8922-d2f6cf75b21b.png">

Reply:
I couldn't find that for the life of me, I thought @Greggless was lying! thanks.

Reply:
Rare occasion when `View source` is more useful then browser developer tools.